core/src/test/java/apoc/schema/SchemasEnterpriseFeaturesTest.java

```diff
@@ -162,8 +163,8 @@ public void testCreateUniqueAndIsNodeKeyConstraintInSameLabel() {
                     .map(record -> (String) record.asMap().get("createStatement"))
                     .collect(Collectors.toList());
             List<String> expectedDescriptions = List.of(
-                    "FOR (galileo:`Galileo`) REQUIRE (galileo.`newton`, galileo.`tesla`) IS NODE KEY",
-                    "FOR ( galileo:`Galileo` ) REQUIRE (galileo.`curie`) IS UNIQUE");
+                    "FOR (n:`Galileo`) REQUIRE (n.`newton`, n.`tesla`) IS NODE KEY",
+                    "FOR (n:`Galileo`) REQUIRE (n.`curie`) IS UNIQUE");
```

Comment:
Any reason why we had to change this?

Reply:
Because previously there was `call db.constraints ()`, now there is `SHOW CONSTRAINTS YIELD createStatement` (line 160). The latter returns a different result.